apps/election-manager/src/components/ImportCVRFilesModal.tsx

```diff
@@ -58,53 +65,76 @@ function throwBadStatus(s: never): never {
   throw new Error(`Bad status: ${s}`)
 }
 
-const ImportCVRFilesModal: React.FC<Props> = ({ isOpen, onClose }) => {
+const ImportCVRFilesModal: React.FC<Props> = ({
+  isOpen,
+  onClose: onCloseFromProps,
+}) => {
   const {
     usbDriveStatus,
     saveCastVoteRecordFiles,
     castVoteRecordFiles,
     electionDefinition,
   } = useContext(AppContext)
-  const [isLoading, setIsLoading] = useState(false)
+  const [currentState, setCurrentState] = useState<ModalState>(ModalState.INIT)
```

Comment:
The type parameter should be inferred properly.

```suggestion
  const [currentState, setCurrentState] = useState(ModalState.INIT)
```

Reply:
Ah yeah I wasn't sure if it would just infer it as a string or not without that

Reply:
Yeah, it handles `enum` values differently from regular strings.